### ISSUE TITLE
Cannot use 'ScheduledInstallation' on xWindowsUpdateAgent on a client OS (Windows 10)

### ISSUE THREAD

**Issue by nebffa** [PowerShell/xWindowsUpdate#27]
```
xWindowsUpdateAgent 'Windows Updates'
    {
        IsSingleInstance = 'Yes'
        UpdateNow = $true
        Category = @('Security','Important')
        Source = 'WindowsUpdate'
        Notifications = 'ScheduledInstallation'
    }
```

yields the following results

```
VERBOSE: [ENETT-NB086]: LCM:  [ Start  Set      ]
VERBOSE: [ENETT-NB086]: LCM:  [ Start  Resource ]  [[xWindowsUpdateAgent]Windows Updates]
VERBOSE: [ENETT-NB086]: LCM:  [ Start  Test     ]  [[xWindowsUpdateAgent]Windows Updates]
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] TryGet RebootRequired...
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Got rebootRequired: False
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Searching for updating using: IsAssigned=1 and IsHidden=0 and IsInstalled=0
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Get default search service: 3da21691-e39d-4da6-8a4b-b43877bcb1b7
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] updateNow compliant: True
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] notifications compliant: False
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] service compliant: False
VERBOSE: [ENETT-NB086]: LCM:  [ End    Test     ]  [[xWindowsUpdateAgent]Windows Updates]  in 5.4950 seconds.
VERBOSE: [ENETT-NB086]: LCM:  [ Start  Set      ]  [[xWindowsUpdateAgent]Windows Updates]
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] TryGet RebootRequired...
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Got rebootRequired: False
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Searching for updating using: IsAssigned=1 and IsHidden=0 and IsInstalled=0
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Get default search service: 3da21691-e39d-4da6-8a4b-b43877bcb1b7
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] updateNow compliant: True
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] notifications compliant: False
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] service compliant: False
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Performing the operation "Set-TargetResource" on target "Set notifications to: ScheduledInstallation".
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Cannot validate argument on parameter 'notificationLevel'. The argument "ScheduledInstallation" does not belong to the set "Not
Configured,Disabled,Notify before download,Notify before installation,Scheduled installation" specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Performing the operation "Set-TargetResource" on target "Disable Microsoft Update Service".
VERBOSE: [ENETT-NB086]:                            [[xWindowsUpdateAgent]Windows Updates] Disable the Microsoft Update setting
```

Note the 

```
The argument "ScheduledInstallation" does not belong to the set "Not
Configured,Disabled,Notify before download,Notify before installation,Scheduled installation" specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.
```

**Comment by pmooij** [comment]
[ValidateSet("Disabled","ScheduledInstallation")] 
differs from 
[ValidateSet('Not Configured','Disabled','Notify before download','Notify before installation','Scheduled installation')]

please align

**Comment by johnf** [comment]
#30 should help

**Comment by johlju** [comment]
Note to anyone in the community wanting to resolve this. PR #30 was closed before it could be merged (and resolve the review comments). So anyone in the community is welcome to run with this.

**Comment by johnf** [comment]
This can be closed it ended up being fixed in #36 which was merged

**Comment by johlju** [comment]
@johnf Thank you for the update! 

I close this as per previous comment.
